### ISSUE TITLE
false-positive for `PD011` and Django's TextChoices

### ISSUE THREAD

**Issue by Zerotask** [astral-sh/ruff#11858]
A call with `ruff check .` and the `PD` rule activated, prints the error 
```
PD011 Use `.to_numpy()` instead of `.values`
```

The mentioned line is `if value not in User.Gender.values:`

`User.Gender` is a [TextChoices](https://github.com/django/django/blob/2a32b233822683c51e59722b7c9aa0789fc4ab1b/django/db/models/enums.py#L107) based class from django which itself inherits from `enum.Enum` and `StrEnum`.
No pandas or numpy involved

**Comment by charliermarsh** [comment]
Thanks, I believe this is the same as #6432.
